themes-default/slim/src/store/modules/notifiers.js

```diff
@@ -124,6 +124,14 @@ const state = {
         authToken: null,
         device: null
     },
+    join: {
```

Comment:
You'll have to remove this file now. As it has been moved to `modules/notifiers/join.js`

Reply:
Will do